forward_engineering/api.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const _ = require('lodash');
```

Comment:
@chulanovskyi-bs please try to not use lodash as a whole and import just the functions you use :)

Reply:
agree, updated